sources/Cargo.toml

```
@@ -76,5 +76,142 @@ members = [
+constants = { version = "0.1", path = "constants" }
+datastore = { version = "0.1", path = "api/datastore" }
+dogtag = { version = "0.1", path = "dogtag" }
+early-boot-config-provider = { version = "0.1", path = "early-boot-config/early-boot-config-provider" }
+generate-readme = { version = "0.1", path = "generate-readme" }
+imdsclient = { version = "0.1", path = "imdsclient" }
+models = { version = "0.1", path = "models" }
+parse-datetime = { version = "0.1", path = "parse-datetime" }
+retry-read = { version = "0.1", path = "retry-read" }
+signpost = { version = "0.1", path = "updater/signpost" }
+storewolf = { version = "0.1", path = "api/storewolf" }
+simple-settings-plugin = { version = "0.1", path = "api/simple-settings-plugin" }
+systemd-derive = { version = "0.1", path = "netdog/systemd-derive" }
+thar-be-updates = { version = "0.1", path = "api/thar-be-updates" }
+update_metadata = { version = "0.1", path = "updater/update_metadata" }
+schnauzer = { version = "0.1", path = "api/schnauzer" }
+
+abi_stable = "0.11.3"
+actix = { version = "0.13", default-features = false }
+actix-rt = "2"
+actix-web = { version = "4", default-features = false }
+actix-web-actors = { version = "4", default-features = false }
+argh = "0.1"
+async-trait = "0.1"
+aws-config = "1"
+aws-sdk-cloudformation = "1"
+aws-sdk-ec2 = "1"
+aws-sdk-eks = "1"
+aws-smithy-runtime = "1"
+aws-smithy-types = "1"
+aws-types = "1"
+bit_field = "0.10"
+bytes = "1"
+cached = "0.49"
+cargo-readme = "3"
+chrono = { version = "0.4", default-features = false }
+darling = { version = "0.20", default-features = false }
+dns-lookup = "2"
+env_logger = "0.11"
+envy = "0.4"
+flate2 = { version = "1", default-features = false }
+fs2 = "0.4"
+futures = { version = "0.3", default-features = false }
+futures-channel = { version = "0.3", default-features = false }
+futures-core = "0.3"
+futures-util = { version = "0.3", default-features = false }
+glob = "0.3"
+gptman = { version = "1", default-features = false }
+handlebars = "4"
+headers = "0.3"
+hex-literal = "0.4"
+http = "0.2"
+httparse = "1"
+httptest = "0.15"
+hyper = { version = "0.14", default-features = false }
+hyper-rustls = { version = "0.24", default-features = false }
+hyper-unix-connector = "0.2"
+indexmap = "1"
+ipnet = "2"
+itertools = "0.13"
+lazy_static = "1"
+libc = "0.2"
+log = "0.4.21"
+lz4 = "1"
+maplit = "1.0"
+nix = "0.26"
+num = "0.4"
+num-derive = "0.4"
+num-traits = "0.2"
+num_cpus = "1"
+pentacle = "1"
+percent-encoding = "2"
+pest = "2.5"
+pest_derive = "2.5"
+proc-macro2 = "1"
+quick-xml = "0.26"
+quote = "1"
+rand = { version = "0.8", default-features = false }
+regex = "1"
+reqwest = { version = "0.11", default-features = false }
+semver = "1"
+serde = "1"
+serde-xml-rs = "0.6"
+serde_json = "1"
+serde_plain = "1"
+serde_yaml = "0.9"
+shell-words = "1"
+shlex = "1"
+signal-hook = "0.3"
+simplelog = "0.12"
+snafu = { version = "0.8", default-features = false }
```

If we specify it this way to enable default features, wouldn't that simplify crate level usage?
```suggestion
snafu = "0.8"
```

Seems like everything except `signpost` uses the default features, and `signpost` should be fine with that instead.